npm/create-cypress-tests/scripts/examples/cypress/support/e2e.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/e2e.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
```

Comment:
I wonder if we should deprecate this library, it has a total for 44 weekly downloads: https://www.npmjs.com/package/create-cypress-tests?activeTab=versions